pkg/services/sqlstore/migrations/migrations.go

```diff
@@ -157,4 +157,17 @@ func addStarMigrations(mg *Migrator) {
 
 	mg.AddMigration("create star table", NewAddTableMigration(starV1))
 	mg.AddMigration("add unique index star.user_id_dashboard_id", NewAddIndexMigration(starV1, starV1.Indices[0]))
+	mg.AddMigration("Add column dashboard_uid in star", NewAddColumnMigration(starV1, &Column{
+		Name: "dashboard_uid", Type: DB_Text, Nullable: true,
+	}))
+	mg.AddMigration("Add column org_id in star", NewAddColumnMigration(starV1, &Column{
+		Name: "org_id", Type: DB_BigInt, Nullable: false, Default: "1",
+	}))
+	mg.AddMigration(
```

Comment:
while we are hear, can we add an updated time field?  then we can use if for the rV

Reply:
added :)